.github/workflows/solaris.yaml

```diff
@@ -41,19 +40,19 @@ jobs:
     runs-on: ubuntu-latest
     name: Test JDK 11, Solaris SPARC
     steps:
-    - name: Test in Solaris SPARC
```

Comment:
Hi So sorry for this, I want to apologize because of this mess, first time being a contributor, thanks a lot for this steps, thanks to you I now more stuff about git!! Will fix my commit with this valuable information